s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
That would be inconsistent with the `apache-{spark,hadoop}-{master,worker}` images. I personally don't understand the motivation for having the commit hash in the tag if we are not building from source`*`.

`*` Actually, even if we're building from source, I don't understand the motivation unless we're specifically checking out a commit as we do in the `adam` image. If you're building a docker image from a tag, then don't use the commit in the docker image tag, as that is redundant.